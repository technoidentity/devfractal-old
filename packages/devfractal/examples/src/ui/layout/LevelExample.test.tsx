import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Level } from './LevelExample'

test('Level Example', () => {
  const component: ReactTestRenderer = renderer.create(<Level />)
  expect(component.toJSON()).toMatchSnapshot()
})
