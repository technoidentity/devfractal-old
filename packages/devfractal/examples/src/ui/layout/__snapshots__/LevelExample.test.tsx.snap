// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Level Right
    </h1>
    <nav
      className="level"
    >
      <div
        className="level-item level-right"
      >
        <div
          className="level-item"
        >
          <strong>
            All
          </strong>
        </div>
        <div
          className="level-item"
        >
          <a
            href="#!"
          >
            Published
          </a>
        </div>
        <div
          className="level-item"
        >
          <a
            href="#!"
          >
            Drafts
          </a>
        </div>
        <div
          className="level-item"
        >
          <a
            href="#!"
          >
            Deleted
          </a>
        </div>
        <div
          className="level-item"
        >
          <div
            className="control"
          >
            <button
              className="button is-success"
              type="button"
            >
              New
            </button>
          </div>
        </div>
      </div>
    </nav>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Level left
    </h1>
    <nav
      className="level"
    >
      <div
        className="level-item level-left"
      >
        <div
          className="level-item"
        >
          <h1
            className="subtitle is-5"
          >
            123 posts
          </h1>
        </div>
        <div
          className="level-item"
        >
          <div
            className="field has-addons"
          >
            <div
              className="control"
            >
              <input
                className="input"
                placeholder="Find a Post"
                type="text"
              />
            </div>
            <div
              className="control"
            >
              <button
                className="button"
                type="button"
              >
                Search
              </button>
            </div>
          </div>
        </div>
      </div>
    </nav>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Centered level
    </h1>
    <nav
      className="level"
    >
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Tweets
          </div>
          <h1
            className="title"
          >
            3,456
          </h1>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Following
          </div>
          <h1
            className="title"
          >
            123
          </h1>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Followers
          </div>
          <h1
            className="title"
          >
            456K
          </h1>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Likes
          </div>
          <h1
            className="title"
          >
            789
          </h1>
        </section>
      </div>
    </nav>
    <nav
      className="level"
    >
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered has-text-info"
          >
            <a
              href="#!"
            >
              Home
            </a>
          </div>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered has-text-info"
          >
            <a
              href="#!"
            >
              Menu
            </a>
          </div>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            <img
              alt="bulma logo"
              src="https://bulma.io/images/bulma-type.png"
              style={
                Object {
                  "height": "30px",
                }
              }
            />
          </div>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered has-text-info"
          >
            <a
              href="#!"
            >
              Reservations
            </a>
          </div>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered has-text-info"
          >
            <a
              href="#!"
            >
              Contact
            </a>
          </div>
        </section>
      </div>
    </nav>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Mobile level
    </h1>
    <nav
      className="level is-mobile"
    >
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Tweets
          </div>
          <h1
            className="title"
          >
            3,456
          </h1>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Following
          </div>
          <h1
            className="title"
          >
            123
          </h1>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Followers
          </div>
          <h1
            className="title"
          >
            456K
          </h1>
        </section>
      </div>
      <div
        className="level-item"
      >
        <section
          className="section"
        >
          <div
            className="has-text-centered"
          >
            Likes
          </div>
          <h1
            className="title"
          >
            789
          </h1>
        </section>
      </div>
    </nav>
  </section>,
]
`;
